tools/ramdump/HowToUseRamdump.md

```diff
@@ -30,7 +30,7 @@ Debug Options -> Enable Debug Output Features to y
 Debug Options -> Enable backtracking using Frame pointer register  to y
 ```
 
-## How to upload RAMDUMP
+## How to upload RAMDUMP: (Linux)
```

Comment:
Could you check line 7? Because you modify the title, link has to be modified together.
How about below?
```
## How to upload RAMDUMP
### In Linux

### In Windows
```

Reply:
modified